docs/sdk/src/guides/your_first_node.rs

```diff
@@ -1 +1,311 @@
 //! # Your first Node
+//!
+//! In this guide, you will learn how to run a runtime, such as the one created in
+//! [`your_first_runtime`], in a node. Within the context of this guide, we will focus on running
```

Comment:
Do we already have docs about the general relationship between the WASM runtime and the node? Like explaining the separation a bit. We could link that here if it exists.